scripts/check_ipfs_hashes_pushed.py

```diff
@@ -33,8 +33,8 @@
 def check_ipfs_hash_pushed(ipfs_hash: str) -> bool:
     """Check that the given ipfs hash exists in the registry"""
     try:
-        url = f"{IPFS_ENDPOINT}/{ipfs_hash}"
-        res = requests.get(url, timeout=5)
+        url = f"{IPFS_ENDPOINT}/{ipfs_hash.strip()}"
+        res = requests.get(url, timeout=120)
```

Comment:
Test will still fail as one package cannot be uploaded currently - > opened a ticket on Trello